### ISSUE TITLE
Sequel Check?

### ISSUE THREAD

**Issue by Aryk** [sportngin/okcomputer#135]
Would you guys be ok to add a check for Sequel.

This should work for most use cases:

```ruby

module OkComputer
  class SequelCheck < Check
    attr_reader :sequel_database, :migration_directory


    def initialize(sequel_database, migration_directory)
      @sequel_database = sequel_database
      @migration_directory = migration_directory
    end

    # Public: Return the schema version of the database
    def check
      mark_message "Schema is #{'not ' unless is_current?}up to date"
    rescue ConnectionFailed => e
      mark_failure
      mark_message "Error: '#{e}'"
    end

    # Public: The scema version of the app's database
    #
    # Returns a String with the version number
    def is_current?
      ::Sequel.extension(:migration)
      ::Sequel::Migrator.is_current?(sequel_database, migration_directory)
    rescue => e
      raise ConnectionFailed, e
    end

    ConnectionFailed = Class.new(StandardError)
  end
end
```

Then, user can do:

```ruby
OkComputer::Registry.register "database", OkComputer::SequelCheck.new(::Sequel::Model.db, 'db/migrate')
```

Thoughts? A variation is you can put the `::Sequel.extension(:migration)` at the top of the file. All it does is require one file.

I would fork, clone, pr and do all that, just thought this was relatively straightforward and maybe could avoid the extra work :)

**Comment by anfleene** [comment]
Sure, we would accept that patch. 👍

**Comment by Aryk** [comment]
@anfleene patch submitted. :)

**Comment by elliott-beach** [comment]
I think this can be closed as #136 was merged.

**Comment by pbyrne** [comment]
Closed by https://github.com/sportngin/okcomputer/pull/136.
